apps/io-session-manager-internal/src/services/__tests__/blocked-users-service.test.ts

```diff
@@ -0,0 +1,43 @@
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+
+import { BlockedUsersService } from "../blocked-users-service";
+
+import { RedisClientTaskMock } from "../../__mocks__/repositories/redis.mock";
+import {
+  mockUnsetBlockedUser,
+  BlockedUsersRedisRepositoryMock,
+} from "../../__mocks__/repositories/blocked-users-redis.mock";
+import { aFiscalCode } from "../../repositories/__tests__/blocked-users-redis.test";
+
+describe("Blocked Users Service", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  const deps = {
+    fastRedisClientTask: RedisClientTaskMock,
+    safeRedisClientTask: RedisClientTaskMock,
+    blockedUserRedisRepository: BlockedUsersRedisRepositoryMock,
+  };
+
+  it("should succeed with an $title session", async () => {
```

Comment:
was it an `it.each` test? I see `$title`

Reply:
Good catch, done in [69811ca](https://github.com/pagopa/io-auth-n-identity-domain/pull/296/commits/69811caa545b2a0e0e3567a479f0dc0ba6d4fd7a)